### PR TITLE
Bur 43 frontend create responsive header

### PR DESCRIPTION
**Pull request by driesvanspauwen**

### CODE REVIEW

File: frontend/src/components/header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                <div className="hidden md:flex md:gap-x-8">
+                    {navigation.map((item) => (
+                        <a key={item.name} href={item.href} className="text-sm font-semibold leading-6 text-gray-900">
+                            {item.name}
+                        </a>
+                    ))}
+                    {/* Mobile menu toggle button */}
+                    {/*TODO: move logic to global state (redux or react context)*/}
+                    {isAuthenticated ?
+                        <a href="profile" className="text-sm font-semibold leading-6 text-gray-900">
```
Comment (rserry) [inline frontend/src/components/header/Header.tsx:80]: redirect to /profile or /account?

File: frontend/src/components/header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+
+                {/* Whitespace and login/profile links */}
+                <div className="hidden md:flex md:gap-x-8">
+                    {navigation.map((item) => (
+                        <a key={item.name} href={item.href} className="text-sm font-semibold leading-6 text-gray-900">
+                            {item.name}
+                        </a>
+                    ))}
+                    {/* Mobile menu toggle button */}
```
Comment (rserry) [inline frontend/src/components/header/Header.tsx:77]: This comment is not needed here I think?

File: frontend/src/components/header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                                        href={item.href}
+                                        className="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50"
+                                    >
+                                        {item.name}
+                                    </a>
+                                ))}
+                            </div>
+                            <div className="py-6">
+                                <a
+                                    href="login"
```
Comment (rserry) [inline frontend/src/components/header/Header.tsx:134]: No check if user is already authenticated, like the desktop version

File: frontend/src/components/login/LoginForm.tsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ import LitusOAuthButton from "@/components/login/LitusOAuthButton";
 import {ApiClient, ApiClientError} from "@/utils/api";
 import {useRouter} from "next/navigation";
 import {SetJWTAsCookie} from "@/utils/oauth";
+import Logo from "@/components/branding/Logo";
```
Comment (rserry) [inline frontend/src/components/login/LoginForm.tsx:10]: unused

**Review by rserry — CHANGES_REQUESTED** [review]

File: frontend/src/components/header/Header.tsx (hunk 1 above)
Comment (driesvanspauwen) [reply frontend/src/components/header/Header.tsx:80]: account is better

**Comment by driesvanspauwen** [comment]
comments are resolved, waiting for BUR-168 to release

**Review by rserry — APPROVED** [review]

**Pull request merged**